docs/supported_profilers/diamond.md

```diff
@@ -0,0 +1,13 @@
+# DIAMOND
+
+> [DIAMOND](https://github.com/bbuchfink/diamond) is a sequence aligner for protein and translated DNA searches, designed for high performance analysis of big sequence data.
+
+## Profile Format
+
+Taxpasta expects a tab-separated file with three columns. Taxpasta will interpret the columns as:
```

Comment:
For all profilers, if there is an option as to which output type to output, we should mention this here.